application/pom.xml

```diff
@@ -33,5 +33,13 @@
             <artifactId>equalsverifier</artifactId>
             <scope>test</scope>
         </dependency>
+
+        <dependency>
+            <groupId>org.assertj</groupId>
+            <artifactId>assertj-core</artifactId>
+            <version>3.25.1</version>
```

Comment:
we have a practice indicate all version numbers in main pom.xml and use dependency management